src/lib/users/clients/auto.rb

```diff
@@ -46,17 +46,23 @@ def initialize
 
       def import(param)
         check_users(param["users"] || [])
-        reader = Y2Users::Autoinst::Reader.new(param)
-        result = reader.read
-        read_linuxrc_root_pwd(result.config)
-
-        if result.issues?
-          return false unless Y2Issues.report(result.issues) == :yes
+        # Use new API for autoinstallation
+        if Yast::Stage.initial
+          reader = Y2Users::Autoinst::Reader.new(param)
+          result = reader.read
+          read_linuxrc_root_pwd(result.config)
+
+          if result.issues?
+            return false unless Y2Issues.report(result.issues) == :yes
+          end
+
+          Y2Users::ConfigManager.instance.register(result.config, as: :autoinst)
```

Comment:
Not related to this PR, but maybe we should start using `target` for the autoyast config too.

Reply:
yep, but as said on call my goal is to not break autoyast UI, so I mainly copy existing stuff without touching it.